azavea/echo-locator
# Stand up new production Amplify stack to fix email verification

## Overview

Fix the bug where users have to perpetually re-verify their email address by standing up a new production Amplify stack with the correct parameters selected.

Closes #42.

## Notes

The diff here is pretty huge, since I renamed the production stack. I'll try to call out any relevant portions of it that relate to things we changed, but in general, I don't expect reviewers to pore over the details of the configuration here. Testing is more important.

## Testing instructions

- Move the production `aws-exports.js` config file into the Taui source directory:

```
cp deployment/amplify/production/aws-exports.js taui/src/aws-exports.js
```

- Start up a development server
  - You can use `./scripts/server` to bring up Docker Compose services, or `npm start` if you're using a local Node installation
- Using the admin user credentials supplied in the `Cognito admin user (production)` LastPass entry, log into http://localhost:9966 and confirm that you don't have to verify your email address
